ta/os_test/os_test.c

```diff
@@ -36,6 +36,14 @@ static TEE_Result check_returned_prop(
 		TEE_Result return_res, TEE_Result expected_res,
 		uint32_t return_len, uint32_t expected_len)
 {
+	/*
+	 * If expecting TEE_ERROR_SHORT_BUFFER but returned size matches
+	 * expected size then TEE_SUCCESS is a valid return code.
+	 */
+	if (expected_res == TEE_ERROR_SHORT_BUFFER &&
+	    return_res == TEE_SUCCESS && return_len <= expected_len)
+		return TEE_SUCCESS;
```

Comment:
Why not change the caller instead? The caller knows if the size is big enough or not.

Reply:
Ok, I agree i would be better.